src/modules/75-cf/components/FlagActivation/__tests__/ServicesList.test.tsx

```diff
@@ -349,4 +348,28 @@ describe('EditServicesModal', () => {
       expect(screen.getByText('cf.featureFlagDetail.noServices')).toBeInTheDocument()
     })
   })
+
+  test('it should render link to take user to create a new Service', async () => {
+    useGetServiceListMock.mockReturnValue({
+      loading: false,
+      data: [],
+      refetch: jest.fn(),
+      error: false
+    } as any)
+
+    renderComponent()
+    userEvent.click(screen.getByRole('button', { name: 'edit-services' }))
+
+    const addNewServiceButton = screen.getByRole('button', { name: 'plus newService' })
+
+    // assert correct empty state
+    await waitFor(() => {
+      expect(screen.getByText('cf.featureFlagDetail.noServices')).toBeInTheDocument()
+      expect(addNewServiceButton).toBeInTheDocument()
+
+      expect(screen.getByRole('link', { name: 'plus newService' }).getAttribute('href')).toContain(
+        '/account/dummy/cv/orgs/dummy/projects/dummy/monitoringservices/setup'
+      )
```

Comment:
You might be able to use `.toHaveAttribute` here
```suggestion
      expect(screen.getByRole('link', { name: 'plus newService' })).toHaveAttribute('href', '/account/dummy/cv/orgs/dummy/projects/dummy/monitoringservices/setup')
```

Reply:
is the benefit just shorter syntax?

Reply:
Does exactly the same thing, so don't feel the need to change. Just reads better than getting a string and asserting against it.

Reply:
noted for next time ✌️